showcase/2021/october/index.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE HTML>
+<html>
+
+<head>
+	<meta charset="utf-8" />
+
+	<!-- Our trusty markdown stylesheet -->
+	<link rel="stylesheet" href="/css/github-markdown.css" />
+	<link rel="stylesheet" href="/css/utils.css" />
+	<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css"
+		integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" crossorigin="anonymous">
+
+	<script src="https://code.jquery.com/jquery-3.5.1.slim.min.js"
+		integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj"
+		crossorigin="anonymous"></script>
+	<script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.1/dist/umd/popper.min.js"
+		integrity="sha384-9/reFTGAW83EW2RDu2S0VKaIzap3H66lZH81PoYlFhbGU+6BZp6G7niu735Sk7lN"
+		crossorigin="anonymous"></script>
+	<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"
+		integrity="sha384-B4gt1jrGC7Jh4AgTPSdUtOBvfO8shuf57BaghqFfPlYxofvL8/KUEfYiJOMMV+rV"
+		crossorigin="anonymous"></script>
+
+	<!-- Load markdeep and set it to not automatically render markdown -->
+	<script>markdeepOptions = { mode: 'script', tocStyle: 'none' };</script>
+	<script src="/scripts/markdeep.min.js"></script>
+
+	<title>October 2021 Showcase</title>
+
+	<!-- Open Graph Metadata -->
+	<meta property="og:title" content="October 2021 Showcase" />
+	<meta property="og:type" content="website" />
+	<meta property="og:url" content="https://graphics-programming.org/showcase/2021/october/" />
+	<meta property="og:image" content="https://imgur.com/JvHWTv6.png" />
+	<meta property="og:description"
+		content="Graphics Programming Community created content showcase for October 2021" />
+</head>
+
+<body>
+	<code style="display:none" id="content">
+		# October 2021 Community Creation Showcase
+
+		<nav class="mx-auto text-center">
+			[Home](/) &bull; [Resources](/resources) &bull; [Showcase](/showcase)
+		</nav>
+
+		<div>
+			<video controls width="100%">
```

Comment:
Use the markup for videos from Imgur.  You can see previous months.

Reply:
Fixed.